doc/source/randindex.rst

```diff
@@ -0,0 +1,33 @@
+Rand indices
+==========================
+
+`Rand index <http://en.wikipedia.org/wiki/Rand_index>`_ is a measure of the similarity between two data clusterings. From a mathematical standpoint, Rand index is related to the accuracy, but is applicable even when class labels are not used.
+
```

Comment:
What does "applicable even when class labels are not used" mean? Does it mean even when the clustering labels do not partition the data, or when there are more than 2 classes? In our case we have "class labels" its the identifier of the cluster to which the data belongs. I looked at the wiki article and was confused by it there too.